salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/context/Context.java

```diff
@@ -11,21 +13,43 @@
  * [MR] Context class to share between Step Definitions.
  */
 public class Context {
+
+    private RequestManager requestManager;
     private List<User> usersList;
 
     /**
      * Constructor for Context class.
      */
     public Context() {
+        requestManager = null;
         this.usersList = new ArrayList<>();
     }
 
+    /**
+     * Sets request specification.
+     *
+     * @param requestSpe
+     */
+    public void setRequestSpec(final RequestSpecification requestSpe) {
+        requestManager.setRequestSpec(requestSpe);
```

Comment:
This line is already present in the Login Step, Is it necesary into this class again??

Reply:
Please, this line is repeated in LoginSteps class, please remove this method from Context